src/hott/06-contractible.rzk.md

```diff
@@ -465,18 +467,18 @@ A type is contractible if and only if it has singleton induction.
 #def contr-implies-singleton-induction-ind
   ( A : U)
   ( is-contr-A : is-contr A)
-  : (has-singleton-induction A)
+  : ( has-singleton-induction A)
   :=
-    ( ( center-contraction A is-contr-A) ,
-      \ B →
+    ( ( center-contraction A is-contr-A)
+    , \ B →
         ( ( \ b x →
                 ( transport A B
                   ( center-contraction A is-contr-A) x
-                  ( realign-homotopy-contraction A is-contr-A x) b)) ,
-          ( \ b →
+                  ( realign-homotopy-contraction A is-contr-A x) b))
```

Comment:
This should be
```suggestion
        ( ( \ b x →
            ( transport A B
              ( center-contraction A is-contr-A) x
              ( realign-homotopy-contraction A is-contr-A x) b))
```
(disregarding the redundant parentheses)

Reply:
Sadly, the indentation has to be adjusted manually for now. There is still some debate over whether the formatter should even enforce indentation or leave it up to the user.